google/cloud/storage/tests/thread_integration_test.cc

```diff
@@ -115,7 +115,7 @@ TEST_F(ThreadIntegrationTest, Unshared) {
   // feature.
   auto const thread_count =
       (std::min)(32U, (std::max)(8U, std::thread::hardware_concurrency()));
-  auto const object_count = 100 * thread_count;
+  auto const object_count = 75 * thread_count;
```

Comment:
If you want to set this to 50 or 25 that would probably work too.

Reply:
OK, 50 it is.  Thanks.